lib_forceload/data/gm4_forceload-1.0/functions/init_chunk.mcfunction

```diff
@@ -3,7 +3,8 @@
 forceload remove 29999999 7134
 forceload add 29999999 7134
 
-fill 29999998 0 7135 29999997 2 7131 minecraft:bedrock
+fill 29999998 0 7135 29999997 2 7130 minecraft:bedrock
 setblock 29999998 1 7134 minecraft:yellow_shulker_box
 setblock 29999998 1 7133 minecraft:birch_wall_sign[facing=north]
 setblock 29999998 1 7132 minecraft:jukebox{RecordItem:{id:"minecraft:stick",Count:1,tag:{}}}
+setblock 29999998 1 7131 minecraft:repeating_command_block{auto:1b,Command:"function #gm4_forceload:command_block_tick"}
```

Comment:
This places the command block in every dimension. However I don't think the current potion tracker `tick` function handles this correctly. Either filter the potions in the current dimension with `x=0`, or only place the repeating command block in the overworld.